sandy/groupr.py

```diff
@@ -197,15 +218,34 @@ def get_flux(self, **kwargs):
         (821000.0, 2231000.0]     1.41000e+06
         (2231000.0, 10000000.0]   7.76900e+06
         Name: iwt, dtype: float64
+
+        >>> endf6 = sandy.get_endf6_file('jeff_33','xs', 922350)
+        >>> groupr = endf6.get_gendf(ek=sandy.energy_grids.CASMO12)
+        >>> groupr.get_flux(mt=4)
```

Comment:
Why does the flux change with the MT?

Reply:
You are right, the flow should not change. It is a mistake to apply the same rule to flux and xs.

Reply:
ok, then should we fix the mistake or not?

Reply:
i see it is fixed now